modules/loader/src/main/java/com/opengamma/strata/loader/LoaderUtils.java

```diff
@@ -234,14 +234,22 @@ public static BigDecimal parseBigDecimal(String str) {
    */
   public static BigDecimal parseBigDecimalPercent(String str) {
     try {
-      return new BigDecimal(str).movePointLeft(2);
+      return new BigDecimal(normalizeIfBracketed(str)).movePointLeft(2);
     } catch (NumberFormatException ex) {
       NumberFormatException nfex = new NumberFormatException("Unable to parse BigDecimal percentage from '" + str + "'");
       nfex.initCause(ex);
       throw nfex;
     }
   }
 
+  private static String normalizeIfBracketed(String value) {
+    if (value.startsWith("(") && value.endsWith(")")) {
+      return "-" + value.replace("(", "").replace(")", "");
```

Comment:
If the input string is "(123(456)" this approach will strip the middle bracket and parse when it shouldn't. You'll need to use `substring()`